fragments/header.sh

```diff
@@ -88,6 +88,13 @@ IGNORE_APP_STORE_APPS=no
 #                  Shouldn’t give any problems for the user in most cases.
 #                  Known bad example: Slack will loose all settings.
 
+# Owner of copied apps
+SYSTEMOWNER=0
```

Comment:
Could `SYSTEMOWNER` be set to `SYSTEMOWNER=1` by default? By setting `SYSTEMOWNER=1` it would make all install scenarios across all computers be equivalent across the board.

Reply:
Or even better we could make a flag that lets users set the value of `SYSTEMOWNER` each time they execute the script.

Reply:
```suggestion
SYSTEMOWNER=1
```

Reply:
No. The history of Installomator is to be like the app was locally installed by the user. You can always sent the variable to the script as a parameter, and it will be set for that run cycle.

Reply:
Okay! Thanks for the suggestion.